src/main/java/htsjdk/tribble/util/ParsingUtils.java

```diff
@@ -100,16 +101,16 @@ public static InputStream openInputStream(String path)
      * @return
      * @throws IOException
      */
-    public static InputStream openInputStream(String path, Function<SeekableByteChannel, SeekableByteChannel> wrapper)
+    public static InputStream openInputStream(final String path, final Function<SeekableByteChannel, SeekableByteChannel> wrapper)
             throws IOException {
 
         final InputStream inputStream;
         if (path.startsWith("http:") || path.startsWith("https:") || path.startsWith("ftp:")) {
             inputStream = getURLHelper(new URL(path)).openInputStream();
-        } else if (IOUtil.hasScheme(path)) {
+        } else if (IOUtil.hasScheme(path) && !IOUtil.getScheme(path).equals("file")) {
             inputStream = new SeekablePathStream(IOUtil.getPath(path), wrapper);
         } else {
-            File file = new File(path);
+            File file = IOUtil.getScheme(path) != null && IOUtil.getScheme(path).equals("file") ? new File(path.substring("file:".length())) : new File(path);
```

Comment:
This branch can be used with the SeekablePathStream too, no?

Reply:
the problem, I think, is that a file can represent a named pipe, which is not seekable.